src/chord_sheet/song.ts

```diff
@@ -419,39 +442,10 @@ Or set the song key before changing key:
     return currentKey;
   }
 
-  /**
-   * Returns a copy of the song with the directive value(s) set to the specified value(s).
```

Comment:
Did you mean to cut down on comment size? I'd vote keeping the code samples in there, they end up in the rendered API docs as examples.